includes/object-cache.php

```diff
@@ -708,6 +709,7 @@ protected function connect_using_phpredis( $parameters ) {
                     $parameters['scheme'],
                     str_replace( 'tls://', '', $parameters['host'] )
                 );
+                $args['ssl'] = $parameters['ssl'];
```

Comment:
Have you tested this with PhpRedis? I don't think this will do anything.

The new constant needs to be added to diagnostics.

SSL isn't really a thing anymore it's all TLS now, should we call it that?

Reply:
'''Setting name'''
Both predis and phpredis's documentation suggest you pass this as 'SSL', as far as I could see. Since you set $parameters directly from the name of the setting, you'd have to have an extra predis line to set 'ssl' from 'tls'. I can do that, but it seems odd.

Also, since this is only SSL options, not any other possible TLS options, it seems odd to name it as TLS. But I can put something in build_parameters() to rename it if you feel WP_REDIS_TLS is better?

'''PhpRedis'''
phpredis docs: https://github.com/phpredis/phpredis#class-redis — shows the use of the 'ssl' key in the arguments, so I'm not sure why it would do nothing to add it?

I've not tested with phpredis, though, because I don't have it installed.

'''Diagnostics'''
Maybe I'm not understanding the code, but $diagnostics gets merged with $parameters in connect_using_predis(), so it will include the 'ssl' value, and in connect_using_phpredis() the setting of $args['ssl'] happens before you add $args to $diagnostics, so I think it's included in either case? Happy to make changes if I'm wrong and you can point me towards what I need to change.

Reply:
Moving to separate threads.